drivers/flash/Kconfig.nrf

```diff
@@ -33,4 +33,13 @@ config SOC_FLASH_NRF_UICR
 	  Enable operations on UICR. Once enabled UICR are written or read as
 	  ordinary flash memory. Erase is possible for whole UICR at once.
 
+config EMULATE_ONE_BYTE_WRITE_ACCESS
+	bool "8-bit write block size emulation"
+	depends on SOC_FLASH_NRF
+	default n
```

Comment:
I wonder whether not better makes this 'y' which keep default behavior as before.
Lets have a pool here: @rakons @pdunaj @ioannisg @lemrey @carlescufi ?

Reply:
I would say no. It adds code, so let's default to no.
@adamkondraciuk please remove the `default n` line, Kconfig bools are `n` by default.

Reply:
Agreed with @carlescufi, however is this considered to be an API change?

Reply:
ok, so agreed. I know that @rakons has the same thought on that.